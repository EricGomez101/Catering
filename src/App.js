import React, { Component } from 'react';
import { Router } from 'buttermilk';
import './App.css';
import Home from './Components/Home/';
import Menu from './Components/Menu';
import Missing from './Components/Missing';

class App extends Component {
  render() {
    return (
      <div className="App">
          <Router
            routes={[
              {
                path: '/',
                render: () => Home,
              },
              {
                path: '/menu',
                render: () => Menu,
              },
              {
                path: '*',
                render: () => Missing,
              }
            ]}
          />
      </div>
    );
  }
}

export default App;
