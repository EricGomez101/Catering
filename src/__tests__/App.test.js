import React from 'react';
import { shallow } from 'enzyme';
import App from '../App';

describe('Components Container', () => {
  let app;
  beforeEach(() => app = shallow(<App/>));
  it("should render correctly in 'debug' mode ", () => {
    expect(app).toMatchSnapshot();
  });

  it("should have a class of name Container", () => {
    
    expect(app.find('div.App')).toHaveLength(1);
  });
});
