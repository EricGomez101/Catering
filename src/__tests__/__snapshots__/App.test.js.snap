// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Components Container should render correctly in 'debug' mode  1`] = `
<div
  className="App"
>
  <Nav />
  <Router
    loadingComponent="div"
    outerComponent="div"
    routeDidChange={[Function]}
    routeWillChange={[Function]}
    routerDidInitialize={[Function]}
    routes={
      Array [
        Object {
          "path": "/",
          "render": [Function],
        },
        Object {
          "path": "*",
          "render": [Function],
        },
      ]
    }
    url=""
  />
</div>
`;
